pkg/operators/common/kubeinventorycache.go

```diff
@@ -83,6 +88,10 @@ func (cache *K8sInventoryCache) Start() {
 
 	// No uses before us, we are the first one
 	if cache.useCount == 0 {
+		cache.factory = informers.NewSharedInformerFactory(cache.clientset, informerResync)
+		cache.pods = cache.factory.Core().V1().Pods().Lister()
+		cache.svcs = cache.factory.Core().V1().Services().Lister()
+
 		cache.exit = make(chan struct{})
 		cache.factory.Start(cache.exit)
 		cache.factory.WaitForCacheSync(cache.exit)
```

Comment:
If a gadget needs the K8sInventoryCache for enrichment this will get instantiated.
After it has finished running everything should be stopped since it is not needed anymore. Keep memory footprint down (I don't know what the factory stores internally. And if it still consumes some amount of cpi time)